documentation/source/library_reference.rst

```diff
@@ -358,16 +358,22 @@ Other Runtime Information
 -------------------------
 
 .. autodata:: cocotb.argv
+   :annotation:
```

Comment:
Does 
```suggestion
   :value:
```

by itself work?

Reply:
Shows this:

`documentation\source\library_reference.rst:360: WARNING: An option to autodata is either unknown or has an invalid value: 'value'`

And the variable doesn't show up in the docs output.